game/static/game/css/forum.css

```
@@ -450,4 +450,317 @@
+html {
+    scroll-behavior: smooth;
+    scroll-padding-bottom: 120px;
+}
+
+.forum-delete-modal {
+    position: fixed;
+    inset: 0;
+    background: rgba(0,0,0,0.65);
+    backdrop-filter: blur(4px);
+
+    display: none;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)